src/components/layout/footer.tsx

```diff
@@ -34,7 +36,9 @@ const Footer = ({
                     {!no_footer_links && (
                         <MainLinksSection is_ppc={is_ppc} is_ppc_redirect={is_ppc_redirect} />
                     )}
-                    <DisclaimerSection is_academy={academy} />
+                    <Branding>
+                        {hide_branding && <DisclaimerSection is_academy={academy} />}
```

Comment:
The purpose of branding is to have this `hide_branding` check inside it.

Reply:
okay